src/patternfly/components/FormControl/examples/form-control-input-example.hbs

```diff
@@ -6,7 +6,7 @@
 {{/form-control}}
 {{#> form-control controlType="input" input="true" type="text" id="textInput4" required="true" aria-invalid="true" aria-label="Error state input example"}}
 {{/form-control}}
-{{#> form-control controlType="input" input="true" pf-c-form-control--modifier="pf-m-form-control-alt" type="text" id="textInput5" aria-label="Alternative input example"}}
+{{#> form-control controlType="input" input="true" form-control--modifier="pf-m-form-control-alt" type="text" id="textInput5" aria-label="Alternative input example"}}
```

Comment:
for these examples we can wrap the attributes in the form-control--attribute=``. you'll have to keep the controlType and convert the required=true, disabled, and readonly.